include/ccf/rpc_context.h

```diff
@@ -140,6 +140,10 @@ namespace ccf
     {
       set_response_trailer(name, std::to_string(n));
     }
+    virtual void set_response_trailer(const http::HeaderKeyValue& kv)
+    {
+      set_response_trailer(kv.first, kv.second);
+    }
```

Comment:
Do we need this overload? We don't have it for `set_response_header`, and it seems weird to make those overload sets different.

Reply:
I added it so that we can call `ctx->set_response_trailer(make_status_trailer(response.status.code()));` as it has to be called in two different places (unary and stream trailers), and I'd forgotten to call `std::to_string(response.status.code())` in one of them, which compiled fine, but caused weird errors at runtime. I'm open to another way of doing this.